csrc/includes/gemm_test.h

```diff
@@ -249,8 +249,8 @@ class StridedGemmTest {
                                         stride_b,
                                         stride_c,
                                         bsz,
-#if defined(__HIP_PLATFORM_AMD__) && TORCH_VERSION_MAJOR <= 2 && TORCH_VERSION_MINOR <=0					
-                                        static_cast<rocblas_gemm_algo>(algo));
+#if defined(__HIP_PLATFORM_AMD__) && TORCH_VERSION_MAJOR <= 2 && TORCH_VERSION_MINOR <=0
+					static_cast<rocblas_gemm_algo>(algo));
```

Comment:
@rraminen Did you intend to introduce tabs instead of spaces?

Reply:
This should be fixed now